runner.go

```diff
@@ -137,6 +138,16 @@ func RunSplunk() bool {
 	return ctx.Err() == nil
 }
 
+func TailFile() bool {
+	args := []string{"-F", os.ExpandEnv(SplunkdLogPath)}
+	tail = exec.CommandContext(ctx, "/usr/bin/tail", args...)
```

Comment:
Possibly change this to a local variable instead of global:
```suggestion
	tail := exec.CommandContext(ctx, "/usr/bin/tail", args...)
```

Reply:
Updated